tests/AppInstanceClientTest.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import os
+import json
+
+from okta import AppInstanceClient
+
+config_path = os.path.normpath(os.path.join(os.path.dirname(__file__),
+                                            '../.sdk.config.json'))
+
+with open(config_path) as sdk_config_data:
+    sdk_config = json.load(sdk_config_data)
+
+
+def build_client(test_description):
+    url = '{}:{}'.format(sdk_config['mockOkta']['proxy'],
+                         sdk_config['mockOkta']['port'])
+    return AppInstanceClient(
+        base_url=url,
+        api_token=sdk_config['mockOkta']['apiKey'],
+        headers={
+            'x-test-description': test_description
+        }
+    )
+
+
+class AppInstanceClientTest(unittest.TestCase):
+
+    def tests_client_initializer(self):
+        client = build_client('/api/v1/events - initialize app instance client')
```

Comment:
great! I like the idea of adding tests for this. The `build_client` method may be a bit of overkill though, and it only allows for one type of instantiation. If we can change this to two tests (one for the `*args` way to instantiate and one for `**kwargs`, that'd be awesome.